osm_fieldwork/OdkCentral.py

```diff
@@ -829,7 +829,7 @@ def getSubmissionPhoto(
         if result.status_code == 200:
             log.debug(f"fetched {filename} from Central")
         else:
-            status = eval(result._content)
+            status = json.loads(result.content)
```

Comment:
Actually, sorry to be a pain, but using `result.json()` from the requests module is more efficient here! It also saves the import of the json module 😄